src/GitHub.VisualStudio/Views/GitHubPane/PullRequestListItem.xaml

```diff
@@ -121,7 +121,7 @@
           </Grid.ColumnDefinitions>
           <Button x:Name="prHashtagLink"
                   Grid.Column="0"
-                  Command="{Binding OpenPROnGitHub, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type local:PullRequestListView}}}"
+                  Command="{Binding DataContext.OpenPullRequestOnGitHub, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type local:PullRequestListView}}}"
```

Comment:
I can see that `DataContext.` is required, but don't understand why. Why is it needed in this context?

Reply:
It's because we're binding to a `RelativeSource`. Usually bindings are relative to the control's `DataContext`, but when you specify a `RelativeSource` the binding will return the control you told it to look for, _not_ that control's `DataContext`. So you need to add the `DataContext` yourself.

Reply:
Previously (before the MVVM refactor) `OpenPROnGitHub` was a command on the view, but the MVVM refactor moved it to the view model (and renamed it) but this binding wasn't updated.